tester/src/test/java/com/silanis/esl/sdk/examples/LocalLanguageExampleTest.java

```diff
@@ -0,0 +1,12 @@
+package com.silanis.esl.sdk.examples;
+
+import org.junit.Test;
+
+public class LocalLanguageExampleTest {
+
+    @Test
+    public void verifyResult() {
+        LocalLanguageExample example = new LocalLanguageExample();
+        example.run();
+    }
```

Comment:
What are you testing here? A test that doesn't do anything makes no sense. Please remove the test.

Reply:
test deleted